ccache/action.yml

```diff
@@ -79,7 +79,7 @@ runs:
         ccache --version || echo "No local ccache installation found"
 
     - name: Setup fixed path ccache caching
-      uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
+      uses: actions/cache@627f0f41f6904a5b1efbaed9f96d9eb58e92e920 # v3.2.4
```

Comment:
Why use 3.2.4 here rather than 4.2.1 as it is below and in the PR description?

Reply:
It seems I have a bug in my tool that automates this. Very sorry let me resolve that.

Reply:
So this is fixed but reviewing it more I think it’s missing a few files.